src/Symfony/Component/HttpKernel/Controller/TraceableControllerResolver.php

```diff
@@ -35,9 +35,11 @@ public function getController(Request $request)
     {
         $e = $this->stopwatch->start('controller.get_callable');
 
-        $ret = $this->resolver->getController($request);
-
-        $e->stop();
+        try {
+            $ret = $this->resolver->getController($request);
+        } finally {
+            $e->stop();
+        }
```

Comment:
The `$ret` variable is no longer necessary with `finally`.
```suggestion
        try {
            return $this->resolver->getController($request);
        } finally {
            $e->stop();
        }
```

Reply:
Good catch, PhpStorm did not even notice!